### DISCUSSION TITLE
How to fix the sorting of bars in bar-chart by metric?

### DISCUSSION THREAD

**Discussion by sw2etheart** [apache/superset#28636] [Q&A / Help]
Hello

As it can be seen from the screenshot, the last group of bars should have been ordered as dark blue -> yellow -> red -> - green -> blue, but the red and yellow are on the wrong positions. I want it to be sorted by the rating (the metric) but it seems to have a non-flexible sorting. Does anybody know how to fix this? Is there a workaround? 
<img width="738" alt="image" src="https://github.com/apache/superset/assets/169693140/9b372e2d-846c-4924-912a-0d88663c260d">

**Comment by sfirke** [comment]
I don't think this is a bug.  Superset makes the assumption that the order of dimension bars should be consistent for each group.  I don't believe there's a way to override this behavior.
